main.tf

```
@@ -49,20 +49,30 @@ resource "google_project_service" "required_apis" {
 
 module "lacework_at_svc_account" {
   source               = "lacework/service-account/gcp"
-  version              = "~> 0.1.0"
+  version              = "~> 1.0"
   create               = var.use_existing_service_account ? false : true
   service_account_name = local.service_account_name
-  org_integration      = var.org_integration
-  organization_id      = var.organization_id
   project_id           = local.project_id
 }
 
 resource "google_storage_bucket" "lacework_bucket" {
-  count         = length(var.existing_bucket_name) > 0 ? 0 : 1
-  project       = local.project_id
-  name          = "${var.prefix}-lacework-bucket-${random_id.uniq.hex}"
-  force_destroy = var.bucket_force_destroy
-  depends_on    = [google_project_service.required_apis]
+  count                       = length(var.existing_bucket_name) > 0 ? 0 : 1
+  project                     = local.project_id
+  name                        = "${var.prefix}-lacework-bucket-${random_id.uniq.hex}"
```

Now that I think about this bucket, our prefix says `lw-at` which means that this
name will look like `lw-at-lacework-bucket-fsa213`. Feels redundant, should we do:
```suggestion
  name                        = "${var.prefix}-${random_id.uniq.hex}"
```

---

Also, I think we have a length limit... 🤔

---

good point! Let's remove this :)